app/views/incidents/_form.html.haml

```diff
@@ -19,8 +19,10 @@
       .field
         = form.label :bus
         = form.text_field :bus, id: :incident_bus
-      .field
+      .field.high-contrast-options
         = form.label :occurred_at
+        - a11y_datetime_labels('incident_occurred_at').each do |label|
```

Comment:
Might be nice to have a helper that spits out both the aria labels and the selects, since you always do one and then the other.